server/sonar-server/src/main/java/org/sonar/server/permission/ws/GroupsAction.java

```diff
@@ -166,6 +145,7 @@ private static WsGroupsResponse buildResponse(List<GroupDto> groups, List<GroupP
   }
 
   private List<GroupDto> findGroups(DbSession dbSession, PermissionQuery dbQuery) {
+    // TODO support organizations
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)